### PR TITLE
Update subscriptions documentation link

### LINKED ISSUE #490: Update the Subscription Watch link for Documentation
## Description
Update the `Documentation` link to the left nav for Subscription Watch. 

## Notes
- Updated documentation location per CCS, link should point towards
   - https://access.redhat.com/documentation/Subscription_Central

## Related issues
- #472 
- https://github.com/RedHatInsights/curiosity-frontend/issues/129

 @jeperry @bclarhk @jlprevatt

### PR DESCRIPTION
**Pull request by cdcabrera**
## What is this
Update the `Documentation` link to the left nav for Subscription Watch based on finalized documentation.

## Environments
- CI
- QA
- Prod Beta

## Note
- link provided by CCS, @jlprevatt

close #490 

@ryelo @karelhala

### CODE REVIEW

File: src/js/App/Sidenav/index.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const documentationLink = {
     rhel: 'https://access.redhat.com/documentation/en-us/cloud_management_services_for_red_hat_enterprise_linux/1.0/',
     insights: 'https://access.redhat.com/documentation/en-us/red_hat_insights/',
     openshift: 'https://docs.openshift.com/container-platform/4.2/',
-    subscriptions: 'https://access.redhat.com/documentation/en-us/red_hat_insights/1.0/'
+    subscriptions: 'https://access.redhat.com/documentation/Subscription_Central'
```
Comment (karelhala) [inline src/js/App/Sidenav/index.js]: This URL throws 404 is this correct URL?

Comment (cdcabrera) [reply src/js/App/Sidenav/index.js]: Indeed @karelhala the docs are currently unavailable from my perspective that's behaving correctly. However, to confirm I'll defer towards 
@jeperry @bclarhk @jlprevatt @rblackbu 
to confirm.

Comment (cdcabrera) [reply src/js/App/Sidenav/index.js]: After some back and forth, looks like we do need to update the PR... the trailing `/` ref needs to be added or it'll continue to `404`. Updating accordingly

**Comment by jlprevatt** [comment]
Adding a comment as proxy for David Voss (CCS DPM) and Matt Werner (CCS CS). Links were added/verified across participants from CCS, ProdMgmt, UX on Dec. 5.

**Pull request merged**